src/context.rs

```diff
@@ -206,7 +208,6 @@ impl<A: Actor> Context<A> {
         // Idk why anyone would do this, but we have to check that they didn't do ctx.stop()
         // in the started method, otherwise it would kinda be a bug
         if !self.check_running(&mut actor).await {
-            self.stop_all();
             actor.stopped().await;
             return;
         }
```

Comment:
I think this should logically go into the `tick` function because that is where this fn is already called and that one already takes `&mut A`. I tried doing that by emitting a special `InboxMessage` if we instantiate `Inbox` while `RunningState::Stopping` but it gets a bit messy.

We do have a test for it now so it should be safe to iterate on some designs!

Reply:
Sounds good to me